static/app/components/dropdownLink.tsx

```diff
@@ -1,7 +1,7 @@
 import {css, useTheme} from '@emotion/react';
 import classNames from 'classnames';
 
-import DropdownMenu from 'sentry/components/dropdownMenu';
+import DropdownMenu from 'sentry/components/dropdownMenuDeprecated';
```

Comment:
Two things here:

* imo, prefixing with `deprecated` will have a bigger impact since we read left to right
* we should also rename the component in the import to include `Deprecated` as well

* I wish we did not allow default exports as it's easy for users to import with any component name :/